cli/ssync/main.go

```diff
@@ -5,13 +5,13 @@ import (
 	"fmt"
 	"os"
 
-	"github.com/rancher/sparse-tools/log"
```

Comment:
Good idea, but I think we need to get this Ssync in first so that Sheng Liang can get it and then do removing of logger for all sparse-tools repo.